yourgov/components/FormMobileFoodVendorPermit/UploadFileTable.tsx

```diff
@@ -68,10 +65,8 @@ export const UploadFileTable = forwardRef<
 		<Stack gap={0.5}>
 			<TableWrapper>
 				<Table
-					aria-rowcount={10}
 					{...(headingId && {
 						'aria-labelledby': headingId,
```

Comment:
Could simplify this to just a normal prop now

Reply:
- [ ] propify